rfcs/models/identity/decentralized-network-identity-discovery-management.md

```diff
@@ -0,0 +1,133 @@
+<!--
+ Copyright IBM Corp. All Rights Reserved.
+
+ SPDX-License-Identifier: CC-BY-4.0
+ -->
+# Decentralized Network-Identity Discovery and Management for Interoperation
+
+* Authors: Venkatraman Ramakrishna, Krishnasuri Narayanam, Bishakh Chandra Ghosh

```

Comment:
You should add Ermyas as an author here as he suggested the primary discovery/advertisement we are using for primary Network DIDs. (Is there any other doc where we talk about discovery?)

Reply:
I have added Ermyas in all relevant docs.